test/e2e/upgrade/common.go

```diff
@@ -269,7 +272,14 @@ func assertNoDegradedCaches() {
 		log, err := testKube.Kubernetes.Logs(provision.InfinispanContainer, pod.GetName(), pod.GetNamespace(), false, ctx)
 		tutils.ExpectNoError(err)
 		if strings.Contains(log, "DEGRADED_MODE") {
-			panic("Detected a cache in DEGRADED_MODE, unable to continue with test")
+			return true
 		}
 	}
+	return false
+}
+
+// healDegradedLocksCache forces the org.infinispan.LOCKS cache to AVAILABLE so that subsequent upgrades can proceed
+func healDegradedLocksCache(operand version.Operand, client tutils.HTTPClient) {
```

Comment:
Do we need this given we opted to start upgrade tests from the  version which should no longer contain LOCKS cache issue? Maybe better keep the code clean

Reply:
That's your call. Skipping the problematic Operands *should* prevent the issue most of the time, but it's possible that we might still encounter the DEGRADED scenario with later 14.x.y Operands. Do you want a more robust testsuite at the expense of not being aware of the underlying issue (we do log it) or do you want to explicitly fail in these scenarios? The .LOCKS cache can become DEGRADED due to network issues as well as the problems we're skipping in the server.

Reply:
I see. Alright, let's keep it here for now as long as we're supporting and testing any 14.0.x versions to on a safe side. Can you add a comment it's a specific for `14.0.x` operands so it's clear that we can remove the code once we move out of 14.0.x completely?